ioc_module.js

```diff
@@ -108,7 +108,7 @@ function registerServices(container) {
 
   container
     .register('ProcessModelService', ProcessModelService)
-    .dependencies('ProcessDefinitionRepository', 'IamService', 'BpmnModelParser');
+    .dependencies('CorrelationService', 'ProcessDefinitionRepository', 'IamService', 'BpmnModelParser');
```

Comment:
I would sort the injected dependencies alphabetically :). 
```suggestion
    .dependencies('BpmnModelParser', 'CorrelationService', 'IamService', 'ProcessDefinitionRepository');
```

Reply:
Aren't all injected dependencies kinda _vital_ for the associated Service? I understand that you can easily see, that the Service depends on its repository, but it also cant work without the other dependencies so I don't really get the advantage when specifying the associated Repository first.